### ISSUE TITLE
Task1: clarification of axis used in BatchNormalization layer

### ISSUE THREAD

**Issue by SyedNaser** [DCASE-REPO/dcase2018_baseline#18]
The input format used in this task is "channel_last", i.e., input(Batch, MelBands, Time, 1), and the batch normalization is performed on "axis=1". The Keras documentation states that with data_format="channel_first" use "axis=1".

I am a bit confused about the axis used in batch normalization.

**Comment by toni-heittola** [comment]
Yes, this is a bug in the parameter file. Correct axis for the batch normalization is of course the last axis, i.e. `axis=-1`. 

Fortunately this will not change results of the system much. Classification accuracy for development dataset was 59.7 +/- 0.7 (over 10 independent trials), and after the axis-fix it is 59.9 +/- 1.3.
